bazel/README.md

```diff
@@ -500,10 +500,11 @@ If you have clang-5.0 or newer, additional checks are provided with:
 bazel test -c dbg --config=clang-asan //test/...
 ```
 
-Similarly, for [thread sanitizer (TSAN)](https://github.com/google/sanitizers/wiki/ThreadSanitizerCppManual) testing:
+[Thread sanitizer (TSAN)](https://github.com/google/sanitizers/wiki/ThreadSanitizerCppManual) tests rely on
+a TSAN-instrumented version of libc++ and have to be run under the docker sandbox:
```

Comment:
There is no way to run tsan without docker now?

Reply:
You should really use instrumented libc++ for that, or weird errors could be reported. It's possible to build an instrumented libc++ and use that locally, but that would require quite a bit more work.

Reply:
How much more work? Is it something other than downloading the libc++ sources and compiling it with some options?

I think that would be my preferred workflow as I've had a lot of trouble with docker.

Reply:
Can we document both options?

Reply:
It is downloading libc++ sources and building it, and then updating bazel config to point to it. 
One way to approach this would be to build libc++ from source as part of TSAN test run (all of this applies to MSAN or any other *SAN tests), but that would increase build time. Perhaps we could have both docker-sandbox and built-at-runtime config options?

Reply:
I would hope this would be part of the flow to set up Envoy development. Maybe optional.

Reply:
We can document / recommend some sort of "standard development environment", how to set up it on workstation / vm / docker, then how to run santizers in it. And not supporting sanitizers in other environment. WDYT?

Reply:
I would probably just doc both options? I think some people might want to build their own libc++.